numpy/linalg/linalg.pyi

```diff
@@ -19,6 +19,8 @@ from numpy import (
     complex128,
 )
 
+from numpy.linalg import LinAlgError as LinAlgError
+
```

Comment:
It looks like something has gone wrong here, this isn't related to lapack_lite

Reply:
Seems https://github.com/numpy/numpy/pull/19909 somehow leaked in here.
I imagine a rebase might fix things.